.github/workflows/test.yml

```
@@ -13,6 +13,7 @@ jobs:
     strategy:
       matrix:
         pgversion: [16, 15, 14, 13, 12]
```

```suggestion
        pgversion: [18, 17, 16, 15, 14, 13, 12]
```
@cyrilgdn, the tests are failing on Pg 17 & 18 on [this GRANT test](https://github.com/cyrilgdn/terraform-provider-postgresql/blob/main/postgresql/resource_postgresql_grant_test.go#L1142), on the reapply the check and validate that there is no diff.
checking the release notes, I didn't find any relevant change on this area.
... also in the `postgresql_grant` resource definition in this repo.